### ISSUE TITLE
Clearing Keyboard shortcuts is not documented

### ISSUE THREAD

**Issue by inopia** [Guake/guake#609]
After updating Guake I found out that keys F1 to F10 got taken by Guake so I needed to unbind them:
![guake-properties](https://cloud.githubusercontent.com/assets/946596/9735969/3384b068-5647-11e5-9573-aee81433ed5e.png)

You might want to add a help text to the bottom on how to clear a bound key. Changing the bind to backspace to clear it wasn't very intuitive. I guess the problem is with the underlying GTK widget.

I tried randomly clicking stuff like delete, escape and modifier keys which didn't work out. I figured it out only after finding this screenshot of gnome-shell:
![gnome-shell](https://cloud.githubusercontent.com/assets/946596/9736123/2569b144-5648-11e5-934c-8edfdc47efa6.jpg)

**Comment by Stibbons** [comment]
Thanks for your report. This is already done in latest version (0.8.0)

**Comment by inopia** [comment]
Great! Keep up the good work!
